src/ResourceManager/Automation/Commands.Automation/Cmdlet/UpdateManagement/RemoveAzureAutomationSoftwareUpdateConfiguration.cs

```diff
@@ -0,0 +1,64 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Azure.Commands.Automation.Common;
+using Microsoft.Azure.Commands.Automation.Properties;
+using System.Management.Automation;
+using System.Security.Permissions;
+using Microsoft.Azure.Commands.Automation.Model.UpdateManagement;
+using System.Globalization;
+
+namespace Microsoft.Azure.Commands.Automation.Cmdlet.UpdateManagement
+{
+    /// <summary>
+    /// Removes a Certificate for automation.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Remove, "AzureRmAutomationSoftwareUpdateConfiguration", SupportsShouldProcess = true,
+        DefaultParameterSetName = AutomationCmdletParameterSets.BySucName)]
```

Comment:
This cmdlet needs an OutputType. It should use [PassThru](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/azure-powershell-design-guidelines.md#returning-no-output) logic.

Reply:
Fixed.